package.json

```diff
@@ -97,23 +97,24 @@
     "prismjs": "1.23.0",
     "reflect-metadata": "0.1.8",
     "rxjs": "^6.5.4",
-    "zone.js": "~0.10.2"
+    "tslib": "^2.0.0",
```

Comment:
@flemcito should this be a devDependency?

Reply:
I believe so, yes - it's tooling for the typescript compiler, so no need for it run time